Detectors/GlobalTracking/include/GlobalTracking/MatchTOF.h

```diff
@@ -241,13 +251,16 @@ class MatchTOF
   //           since this info is provided by external device
   gsl::span<const o2::dataformats::TrackTPCITS> mTracksArrayInp;  ///< input tracks
   std::vector<o2::dataformats::TrackTPCITS>* mTracksArrayInpVect; ///< input tracks (vector to read from tree)
-  std::vector<o2::tpc::TrackTPC> mTPCTracksArrayInp;              ///< input TPC tracks
+  gsl::span<const o2::tpc::TrackTPC> mTPCTracksArrayInp;          ///< input TPC tracks
+  std::vector<o2::tpc::TrackTPC>* mTPCTracksArrayInpVect;         ///< input tracks (vector to read from tree)
   gsl::span<const Cluster> mTOFClustersArrayInp;                  ///< input TOF clusters
   std::vector<Cluster>* mTOFClustersArrayInpVect;                 ///< input TOF clusters (vector to read from tree)
 
   o2::dataformats::MCTruthContainer<o2::MCCompLabel> mTOFClusLabels;                  ///< input TOF clusters MC labels
   o2::dataformats::MCTruthContainer<o2::MCCompLabel>* mTOFClusLabelsPtr;              ///< input TOF clusters MC labels (pointer to read from tree)
   std::vector<o2::MCCompLabel> mTracksLblWork;                                        ///<TPCITS track labels
+  o2::dataformats::MCTruthContainer<o2::MCCompLabel> mTPCTrackLabels;                 ///< input TPC tracks MC labels (TPC only tracks)
+  o2::dataformats::MCTruthContainer<o2::MCCompLabel>* mTPCTrackLabelsPtr;             ///< input TOF clusters MC labels (pointer to read from tree)
```

Comment:
The labels of tracks are provided via ``std::vector<o2::MCCompLabel>``, not ``MCTruthContainer``

Reply:
This data member is no longer needed. I remove it